google-cloud-spanner/src/test/java/com/google/cloud/spanner/IntegrationTestWithClosedSessionsEnv.java

```diff
@@ -86,6 +87,20 @@ PooledSessionFuture getSession() {
       return session;
     }
 
+    @Override
+    SessionFutureWrapper getMultiplexedSession() {
```

Comment:
I don't think this is ever used, as the integration test that uses this test env is (correctly) skipped for multiplexed sessions. And if it is used, what does this then mean?

Reply:
No, this is still used in the regular session call path. We have replaced `getSession()` method with `getMultiplexedSession ` in `DatabaseClientImpl` class for all RO transactions in this PR. So if we don't add this override behaviour, those integration tests will fail.